### PR TITLE
Easier Implementation and user experience improvements

### PR DESCRIPTION
**Pull request by Ri2parna**
The code has been simplified.
The user experience has been improved.

### CODE REVIEW

File: fizzbot.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import requests
+import json
+import os
```
Comment (junnydelacruz) [inline fizzbot.py:3]: no comment

**Review by junnydelacruz — COMMENTED** [review]
noops challenges master

File: fizzbot.py (hunk 1 above)
Comment (Ri2parna) [reply fizzbot.py:3]: I didn't understand what you meant.Can you explain?Is there something wrong with the code?

**Comment by Ri2parna** [comment]
Added Auto fizzbuzz solver.

**Comment by Ri2parna** [comment]
@doolittle ,I added two programs,one auto solver and the other an interactive version. Can they be merged?

**Pull request closed without merging**